docker-compose.yml

```diff
@@ -100,7 +100,7 @@ services:
             file: common.yml
             service: mender-base
         environment:
-            - WORKFLOWS_MONGO_URL=mongodb://mongo-workflows:27017
+            WORKFLOWS_MONGO_URL: mongodb://mender-mongo:27017
```

Comment:
As mentioned on slack, I believe the different syntax is not important:
https://docs.docker.com/compose/compose-file/
You can express env variables both ways.